src/views/admin/PendingCredits.vue

```diff
@@ -87,6 +87,13 @@
           showConfirmButton: false,
           timer: 3000
         })
+      },
+      async updatePendingCreditRequests() {
+        try {
+          await this.$store.dispatch('fetchPendingCreditRequests')
+        } catch (e:any) {
+          console.error('Failed to fetch pending credit requests', e.message + "\nDebug: " + JSON.stringify(e))
+        }
```

Comment:
same remark, a message visible to the end-user was expected.

Reply:
It's handled by the next commit, same as above.

Reply:
Squashed commits. Changed.